polyA/_runners.py

```diff
@@ -198,6 +197,21 @@ def run_full(
             else:
                 stops_matrix.append(tr.stop)
 
+    # Save original alignments for SODA viz
+    subfam_alignments = list(subfamily_sequences_matrix)
+    chrom_alignments = list(chromosome_sequences_matrix)
+    subfam_align_matrix: Dict[str, int] = {}
```

Comment:
I don't see this ever actually used...

Reply:
Yes I'm removing it now - thanks for noticing this!